### PR TITLE
Update copy and add gif to fb-redirect page

### PR DESCRIPTION
**Pull request by cl2205**
#### What's this PR do?

- Updates copy and add gif to fb-redirect page for non-US users
- Updates Squad page with link to an shinetext.advice article on referral tips

#### How was this tested? How should this be reviewed?
Tested locally in desktop and mobile views. Checked that the new link on Squad page was captured in GA.

### CODE REVIEW

File: views/squad.ejs [hunk 1]
```diff
@@ -6,6 +6,14 @@
 <section>
   <p>As a Shine Squad member, you have access to an exclusive community of go-getters who will do just that. Shine Squad members help each other with jobs, advice, passion projects, recognition and a bunch of other stuff you'll only know about if you JOIN.</p>
   <p>Just refer <strong>10 friends</strong> to get your exclusive invite.</p>
+  <p>Need a few ideas to help you refer friends? 
+    <a  
+      href="<%= adviceBaseUrl %>/articles/articles/14-no-fail-tips-to-help-you-refer-friends-and-join-the-shine-squad/?utm_source=Shine&utm_medium=SquadPage"
```
Comment (jonuy) [inline views/squad.ejs]: I think there's an extra `/articles` here

File: api/controllers/WebViewController.js [hunk 2]
```diff
@@ -50,20 +50,34 @@ module.exports = {
 
   ////////////////////////////////////////////////////////////////////////////
 
+  squad: (req, res) => {
```
Comment (jonuy) [inline api/controllers/WebViewController.js]: Moving it here is fine. Just wondering what you're thinking was behind it.

File: views/squad.ejs (hunk 1 above)
Comment (cl2205) [inline views/squad.ejs]: ooh thanks for catching!

File: api/controllers/WebViewController.js (hunk 2 above)
Comment (cl2205) [inline api/controllers/WebViewController.js]: I wasn't able to reference the `adviceBaseUrl` env variable from the `routes` file where this was originally located (the `locals` object was only accepting literal string values from there), so I moved it here in order to be able to pass in that variable.

**Pull request merged**